### PR TITLE
Fix some tests

### PR DESCRIPTION
**Pull request by mabartos**
@rbri I did some refactoring and fixed some tests.

### CODE REVIEW

File: src/main/java/org/openqa/selenium/htmlunit/HtmlUnitWebElement.java [hunk 1]
```diff
@@ -131,6 +133,19 @@ public void submit() {
     parent.submit(this);
   }
 
+  @Override
+  public String getDomAttribute(String name) {
+    //TODO
+    return getAttribute(name);
```
Comment (mabartos) [inline src/main/java/org/openqa/selenium/htmlunit/HtmlUnitWebElement.java:139]: @rbri I'm not really sure about these 2 methods. Do you have any idea how to implement them?

Comment (shs96c) [reply src/main/java/org/openqa/selenium/htmlunit/HtmlUnitWebElement.java:139]: The relevant bit of the spec is here: https://w3c.github.io/webdriver/#get-element-attribute and there are links to the right bits of other specs there, but the TL;DR is that “get element property” is essentially “element.property” in JS.

**Review by rbri — COMMENTED** [review]
It will be great if you can build separate pr's for different refactorings. Otherwise it will be hard do understand the different steps you made.
Will have a look at this later today. Thanks for the PR,

File: src/main/java/org/openqa/selenium/htmlunit/HtmlUnitWebElement.java (hunk 1 above)
Comment (rbri) [reply src/main/java/org/openqa/selenium/htmlunit/HtmlUnitWebElement.java:139]: Will try to write a bunch of test cases for this later today.

**Comment by mabartos** [comment]
@rbri I created a few separate PRs. Thanks

**Pull request closed without merging**